packages/react-query-devtools/src/devtools.tsx

```diff
@@ -418,9 +418,9 @@ export const ReactQueryDevtoolsPanel = React.forwardRef<
 
   const [filter, setFilter] = useLocalStorage('reactQueryDevtoolsFilter', '')
 
-  const [sortDesc, setSortDesc] = useLocalStorage(
+  const [baseSort, setBaseSort] = useLocalStorage(
     'reactQueryDevtoolsSortDesc',
```

Comment:
should we chose a new name here? Otherwise, stored values might be picked up:

```suggestion
    'reactQueryDevtoolsBaseSort',
```